src/microbe_stage/Patch.cs

```diff
@@ -32,15 +32,28 @@ public class Patch
     [JsonProperty]
     private Deque<PatchSnapshot> history = new Deque<PatchSnapshot>();
 
+#pragma warning disable 169 // Used through reflection
+    [JsonProperty]
+    private string untranslatedName;
+#pragma warning restore 169
+
+    public Patch()
```

Comment:
Is this needed? Would the implicitly defined constructor work, or does that not work?

Reply:
It doesn't work with the implicit constructor and throws an exception when trying to load a save: `Newtonsoft.Json.JsonException: no suitable constructor found for current type: Patch`

Reply:
Okay. I think I remember experiencing the same thing. Btw I wrote that constructor finding code, so it might be possible to special case it for calling the implicit constructor.